### PR TITLE
option for low latency mode on linux

### LINKED ISSUE #287: Setting ASYNC_LOW_LATENCY on a serial (on Linux)
Is there a way to set ASYNC_LOW_LATENCY in pyserial? This is useful when using FTDI USB-Serial converters.

Basically, we need to set the flag using ioctl:
```
ioctl(mPhysicalComPort, TIOCGSERIAL, &serial); 
serial.flags |= ASYNC_LOW_LATENCY; // (0x2000)
ioctl(mPhysicalComPort, TIOCSSERIAL, &serial);
```

I guess this ioctl is probably Linux only.

### PR DESCRIPTION
**Pull request by jabdoa2**
fixes #287

### CODE REVIEW

File: serial/serialutil.py [hunk 1]
```diff
@@ -186,6 +186,7 @@ def __init__(self,
                  dsrdtr=False,
                  inter_byte_timeout=None,
                  exclusive=None,
+                 low_latency=None,
```
Comment (zsquareplusc) [inline serial/serialutil.py]: the platform dependent `set_low_latency_mode` would be OK, but i'd prefer to have no changes on the base class as of now.

Comment (jabdoa2) [reply serial/serialutil.py]: I will make that change. Just give me a few minutes

**Comment by jabdoa2** [comment]
@zsquareplusc fixed it.

Did a quick benchmark on a genuine FTDI. Not very scientific but simple:

```
$ python3 -m timeit -s "import serial; ser = serial.Serial('/dev/ttyUSB0'); ser.set_low_latency_mode(False)" -n 1000 "ser.write(b'hello'); ser.read(5)"
1000 loops, best of 3: 16 msec per loop
$ python3 -m timeit -s "import serial; ser = serial.Serial('/dev/ttyUSB0'); ser.set_low_latency_mode(True)" -n 1000 "ser.write(b'hello'); ser.read(5)"
1000 loops, best of 3: 5.99 msec per loop
```

**Comment by zsquareplusc** [comment]
Thank you.

**Comment by jabdoa2** [comment]
@zsquareplusc Would it be possible to get a release with this commit? We would like to use it in the Mission Pinball Framework (https://github.com/missionpinball/mpf/)

**Comment by mje-nz** [comment]
@zsquareplusc This is coming up for us as well, would it be possible to get a release with this commit?

**Comment by navehgrofi** [comment]
@zsquareplusc 
is a new release with this merge expected? we would love to use it and really need it

thanks in advance

**Comment by alexpacini** [comment]
You can install it directly from github using pip:
```shell
python3 -m pip install --user --upgrade git+https://github.com/pyserial/pyserial
```

**Pull request merged**